saucelabs.conf.js

```diff
@@ -5,17 +5,24 @@ const tunnelName = process.env.TUNNEL_IDENTIFIER || 'reformtunnel';
 const waitForTimeout = 60000;
 const smartWait = 5000;
 
+const defaultSauceOptions = {
+  username: process.env.SAUCE_USERNAME,
+  accessKey: process.env.SAUCE_ACCESS_KEY,
+  tunnelIdentifier: process.env.TUNNEL_IDENTIFIER || 'reformtunnel',
+  acceptSslCerts: true,
+  tags: ['idam-web-public']
+};
+
 const getBrowserConfig = browserGroup => {
     const browserConfig = [];
     for (const candidateBrowser in supportedBrowsers[browserGroup]) {
         if (candidateBrowser) {
-            const desiredCapability = supportedBrowsers[browserGroup][candidateBrowser];
-            desiredCapability.acceptSslCerts = true;
-            desiredCapability.tunnelIdentifier = tunnelName;
-            desiredCapability.tags = ['idam-web-public'];
+            console.log("Setting up " + browserGroup)
```

Comment:
You can probably take this out, I think I added this as a debug when it wasn't working